app-android/app/src/main/java/de/tutao/tutanota/credentials/AccountManagerAuthenticator.kt

```diff
@@ -0,0 +1,45 @@
+package de.tutao.tutanota.credentials
+
+import android.accounts.*
+import android.content.Context
+import android.os.Bundle
+
+class AccountManagerAuthenticator(context: Context) : AbstractAccountAuthenticator(context) {
+	private val mContext: Context
```

Comment:
I know it's copypaste but can we please drop hungarian notation here and elsewhere